### ISSUE TITLE
Refresh project list button

### ISSUE THREAD

**Issue by hlung** [RobertGummesson/BuildTimeAnalyzer-for-Xcode#79]
After adding Other Swift Flags and hitting build in Xcode, the new build doesn't appear in the project table. It should automatically detect the change, or add a refresh button and an instruction to refresh if Build Time Analyzer app is opened before the new build.

**Comment by RobertGummesson** [comment]
Yes, there is code to detect it automatically upon build completion but it sounds like it's not working properly.
